src/diskquota_utility.c

```diff
@@ -1700,10 +1699,10 @@ check_hash_fullness(HTAB *hashp, int max_size, const char *warning_message, Time
 	{
 		TimestampTz current_time = GetCurrentTimestamp();
 
-		if (TimestampDifferenceExceeds(*last_overflow_report, current_time,
-		                               diskquota_hashmap_overflow_report_timeout * 1000))
+		if (*last_overflow_report == 0 || TimestampDifferenceExceeds(*last_overflow_report, current_time,
+		                                                             diskquota_hashmap_overflow_report_timeout * 1000))
 		{
-			ereport(WARNING, (errmsg("%s %d", warning_message, guc_value)));
+			ereport(WARNING, (errmsg(warning_message)));
```

Comment:
```suggestion
			ereport(WARNING, (errmsg("%s", warning_message)));
```
to avoid substitution